src/hooks/useSignInDialog/Steps/TermsStep.tsx

```diff
@@ -98,27 +98,31 @@ const TermsStep: React.FC<TermsStepProps> = ({ onStepChange }) => {
             classNames={transitions.names.fade}
             unmountOnExit
           >
-            <ScrollButton onClick={handleScrollToBottom}>
+            <ScrollButton variant="secondary" onClick={handleScrollToBottom}>
               <SvgGlyphChevronDown />
             </ScrollButton>
           </CSSTransition>
         </TermsOverlay>
       </TermsBox>
-      <TermsForm onSubmit={handleSubmit}>
-        <CSSTransition
-          in={isCheckboxVisible}
-          timeout={parseInt(transitions.timings.loading)}
-          classNames={transitions.names.fade}
-          unmountOnExit
-        >
-          <Checkbox
-            value={isAccepted}
-            onClick={() => setIsAccepted(!isAccepted)}
-            label="I’ve read and accept Terms And Conditions"
-          />
-        </CSSTransition>
-        <ContinueButton disabled={!isAccepted}>Continue</ContinueButton>
-      </TermsForm>
+      <StepFooter>
+        <TermsForm onSubmit={handleSubmit}>
```

Comment:
Is there some advantage to this being a form instead of just `div` or something?